app/src/main/java/org/hackillinois/android/view/schedule/DayFragment.kt

```diff
@@ -0,0 +1,95 @@
+package org.hackillinois.android.view.schedule
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.os.Parcelable
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_schedule_day.view.*
+
+import org.hackillinois.android.R
+import org.hackillinois.android.database.entity.Event
+import org.hackillinois.android.viewmodel.ScheduleViewModel
+
+class DayFragment : Fragment() {
+
+    private lateinit var recyclerView: RecyclerView
+    private var adapter: RecyclerView.Adapter<*>? = null
+    private lateinit var layoutManager: RecyclerView.LayoutManager
+    private var sortedEvents: List<Event>? = null
+    private var listState: Parcelable? = null
+
+    override fun onResume() {
+        super.onResume()
+        adapter?.notifyDataSetChanged()
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val sectionNumber = arguments?.getInt(ARG_SECTION_NUM) ?: 0
+
+        val viewModel = ViewModelProviders.of(this).get(ScheduleViewModel::class.java)
+        viewModel.init()
+
+        val liveData = when (sectionNumber) {
+            0 -> viewModel.fridayEventsLiveData
+            1 -> viewModel.saturdayEventsLiveData
+            else -> viewModel.sundayEventsLiveData
```

Comment:
can we put a 2 here or does it have to be an else?

Reply:
Needs to be an else, otherwise it doesn't exhaust all cases.